package.json

```diff
@@ -53,34 +71,53 @@
     "karma-tap-pretty-reporter": "4.2.0",
     "karma-webpack": "git://github.com/ryanclark/karma-webpack#eec6dfc",
     "kw-doc": "2.0.0",
-    "kw-web-suite": "11.0.0",
+    "moment": "2.29.1",
     "node-pre-gyp": "0.16.0",
+    "parallel-webpack": "2.6.0",
+    "postcss-loader": "4.1.0",
+    "prettier": "2.2.1",
     "process": "0.11.10",
+    "raw-loader": "4.0.2",
     "request": "2.88.2",
     "resemblejs": "3.2.5",
+    "semantic-release": "17.3.1",
+    "shader-loader": "1.3.1",
```

Comment:
Should we have the worker/shader loaders in plain dependencies rather than dev? So consumer will benefit from them directly or should we document the fact that they need to add those to their project?

Reply:
Right now we have them included as peer dependencies. Probably worthwhile to stick it in the main dependencies.

Reply:
up to you, but it might be good enough to be in the dep to prevent user error/complain